realm/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="io.realm" >
-    <application/>
+    <application android:allowBackup="true"/>
```

Comment:
This conflicts with anyone that wants to set this value false. As this is a library and not an application, the entire application tag and be removed solving the lint complaint and manifest merge issues.